crypto/src/swap.rs

```diff
@@ -82,14 +231,124 @@ impl From<SwapPlaintext> for pb::SwapPlaintext {
     }
 }
 
+impl From<&SwapPlaintext> for [u8; SWAP_LEN_BYTES] {
+    fn from(swap: &SwapPlaintext) -> [u8; SWAP_LEN_BYTES] {
+        let mut bytes = [0u8; SWAP_LEN_BYTES];
+        bytes[0] = SWAP_TYPE;
+        bytes[1..65].copy_from_slice(&swap.trading_pair.to_bytes());
```

Comment:
Since it's already publicly identified in the `Swap`, do we need to encrypt the trading pair / include it in the `SwapPlaintext`?

Cons: takes extra space, redundant data
Pros: might mean that a client only needs to hold the `SwapPlaintext` to be able to claim the swap, and doesn't necessarily need a full transaction ?

That pro could be pretty good and worth the extra 64 bytes.